mono/metadata/verify.h

```diff
@@ -8,7 +8,6 @@
 #include <mono/metadata/metadata.h>
 #include <mono/metadata/image.h>
 #include <mono/metadata/loader.h>
-#include <mono/metadata/class-internals.h>
 #include <glib.h> /* GSList dep */
```

Comment:
I would say it's fine to break the API in this case? I mean, realistically, nobody was ever able to use this header because we don't install eglib's glib.h as part of Mono's public headers, so I doubt you'd *actually* be breaking anybody's code.